UKHomeOffice/cop-ui
# Added functionality to switch type of form depending on which is received.

### Description
Allows Timeline details to show either a react form or a formIO form depending on which type of form is received.

https://support.cop.homeoffice.gov.uk/browse/COP-10345
### To test
Check that timeline detail is able to render either type of form. This would be done by submitting a react form and then a FormIO form as part of a case to make sure they can both render.

To check that react forms are read-only in a 'check-your-answers' format. There should be no ability to change any of the submitted answers.

### Notes
The formIO version is still using the old functionality for rendering with the default formIO renderer with the fields disabled. There is another ticket in progress still which will allow the formIO forms to look very similar to the react forms so users should not be able to tell which type of form they have received.
